web/app/scripts/src/services/api.es.js

```diff
@@ -17,10 +22,16 @@ export const api = function ($http, PROJECT_SETTINGS) {
     }
 };
 
-export const service = [
-    '$http',
-    'PROJECT_SETTINGS',
-    api
-];
+export const module = {
+    moduleName: 'incuna.surveys-api',
+    componentName: 'API'
+};
+
+angular.module(module.moduleName, [ProjectConfig.moduleName])
+    .service([
+        '$http',
+        ProjectConfig.componentName,
+        API
```

Comment:
Your tests then need to be kept up-to-date with the dependencies required for this definition, and you need to inject the dependencies yourself. As soon as you add another dependency to this, in any position, the tests will fail.

It's way, way simpler to let angular do all that and use angular-mocks' `module()` function to mock it. Adding dependencies to angular components won't make the tests fail. If you want, you can DRY your mocks so updates to this definition then only require updates to tests code in one place rather than several.